app/containers/App.tsx

```diff
@@ -31,6 +31,24 @@ export default function App(props: Props) {
         </Navbar.Group>
 
         <Navbar.Group align={Alignment.RIGHT}>
+          <div className={styles.externalLinks}>
+            <a
+              href="https://github.com/Appsilon/mbaza/discussions"
+              target="_blank"
+              rel="noreferrer"
+              className={styles.link}
+            >
+              Forum
+            </a>
+            <a
+              href="https://github.com/Appsilon/mbaza"
```

Comment:
We have user guides in two languages (English, French) in PDF format; the links are listed in the "Useful files" section of the README. When Robbie requested this change, I think he wanted us to link directly to those files.

We should either link both the English and French versions, or change the link depending on the selected language.

Reply:
I have updated this in the PR